sync/config.py

```diff
@@ -151,26 +151,11 @@ def get_config() -> Configuration:
 
 def __getattr__(name):
     if name == "CONFIG":
-        global _config
-        if _config is None:
-            _config = Configuration()
-        return _config
+        return get_config()
```

Comment:
Linter was complaining about the complexity of this function, so just moved these all out into separate getters

Reply:
Interesting. What caused it to start complaining?

Reply:
I'm not sure, to be honest! Just popped up when I ran the linter locally